File: android/src/test/resources/com/dropbox/android/java/FakeIgnoredMethodTest.java
/*
 * Copyright (c) 2018, Dropbox, Inc. All rights reserved.
 */

package com.dropbox.android.java;

import android.support.test.runner.AndroidJUnit4;
import com.dropbox.core.test.ui_runner.annotations.UiTest;
import org.junit.Ignore;
import org.junit.Test;
import org.junit.runner.RunWith;

/**
 * A class with ignored methods, parser should find 1 valid test
 */
@UiTest
@RunWith(AndroidJUnit4.class)
public class FakeIgnoredMethodTest {
    /**
     * lint needs a javadoc comment
     */
    @Test
    public void emptyTest1() {}

    /**
     * lint needs a javadoc comment
     */
    @Ignore
    @Test
    public void emptyTest2() {}
}

File: src/test/resources/com/dropbox/android/kotlin/FakeIgnoredMethodTest.kt
/*
 * Copyright (c) 2018, Dropbox, Inc. All rights reserved.
 */

package com.dropbox.android.kotlin

import android.support.test.runner.AndroidJUnit4
import com.dropbox.core.test.ui_runner.annotations.UiTest
import jdk.nashorn.internal.ir.annotations.Ignore
import kotlin.test.Test
import org.junit.Ignore
import org.junit.runner.RunWith

/**
 * A class with ignored methods, parser should find 1 valid test
 */
@UiTest
@RunWith(AndroidJUnit4::class)
class FakeIgnoredMethodTest {
    /**
     * lint needs a javadoc comment
     */
    @Test
    fun emptyTest1() {
    }

    /**
     * lint needs a javadoc comment
     */
    @Ignore
    @Test
    fun emptyTest2() {
    }

    /**
     * lint needs a javadoc comment
     */
    @Test
    fun emptyTest3() {
    }
}
